client/src/app/App.js

```diff
@@ -378,7 +378,11 @@ export class App extends PureComponent {
       const { button } = await this.showCloseFileDialog({ name });
 
       if (button === 'save') {
-        await this.saveTab(tab);
+        var saved = await this.saveTab(tab);
```

Comment:
Could make this a `const` to comply with our ES6 standards in the Camunda Modeler.

Reply:
Also, `var` will be hoisted, while `const` won't.

Reply:
I agree, we should use const. I created a follow-up PR: https://github.com/camunda/camunda-modeler/pull/2393

Reply:
If we want to enforce it, shall we use [`eslint`](https://eslint.org/docs/rules/no-var) for that?